### PR TITLE
fix: remove box list

### PR DESCRIPTION
**Pull request by vincentaudebert**
## Type

- Refactor

#### What is expected?

Removes box from list

### CODE REVIEW

File: src/components/ActionBar/index.tsx [hunk 1]
```diff
@@ -17,6 +17,8 @@ const StyledDiv = styled.div<{ rank: number }>`
   transform: translate(-50%, 0);
   width: 600px;
   z-index: 2;
+  display: flex;
+  align-items: center;
```
Comment (chambo-e) [inline src/components/ActionBar/index.tsx:21]: intended ?

File: src/components/List/index.tsx [hunk 2]
```diff
@@ -56,6 +55,8 @@ export type ListBodyRenderProps<DataType> = {
 
 type BodyProps<DataType> = {
   children: (props: ListBodyRenderProps<DataType>) => JSX.Element
+  className?: string
+  style?: React.CSSProperties
```
Comment (matthprost) [inline src/components/List/index.tsx:59]: We don't do `React.XX` anymore, you can import `CSSProperties` directly

**Review by matthprost — CHANGES_REQUESTED** [review]

File: src/components/ActionBar/index.tsx (hunk 1 above)
Comment (vincentaudebert) [reply src/components/ActionBar/index.tsx:21]: yes select bar is broken otherwise :/

[the remark this answers is not in the document]
Comment (vincentaudebert) [reply src/components/ActionBar/index.tsx]: good point. fixed

**Review by chambo-e — APPROVED** [review]

**Review by philibea — APPROVED** [review]

**Review by matthprost — APPROVED** [review]

**Pull request merged**